### ISSUE TITLE
Informix and Python 3.8.1

### ISSUE THREAD

**Issue by ewmadrid** [OpenInformix/IfxPy#28]
After updating Slackware it gave me a new Python version 3.8.1
Now my python scripts fails.
I ran the upgrade command:
pip3 install ifxpy --upgrade
ERROR: Could not find a version that satisfies the requirement ifxpy (from versions: none)
ERROR: No matching distribution found for ifxpy
I get the above error...

On the https://pypi.org/project/IfxPy/ site it only shows these available options:
Programming Language
Python :: 2.7
Python :: 3.4
Python :: 3.5
Python :: 3.6
Python :: 3.7
Python :: Implementation :: CPython

It looks like the Python :: 3.8 version is not available...

I also had the same issue with a script that talks to a MongoDB database.
I ran the upgrade command "pip3 install pymongo --upgrade" and it fixed the problem.

Is Informix support being ended or just not a priority?

Thanks for any help you can provide.
Eric

**Comment by deokershesh** [comment]
@ewmadrid, We will update the Informix Python Driver repository soon with Python 3.8 binaries. Meanwhile, You can also compile the code using Python 3.8 version and use the same. By the what all platforms (Windows, Linux etc) you are looking for?

**Comment by ewmadrid** [comment]
Linux, Slackware 14.2 x86_64 (post 14.2 -current)

**Comment by ewmadrid** [comment]
Linux:
Slackware 14.2 x86_64 (post 14.2 -current)
Debian GNU/Linux 10.2
CᴇntOS Linux 8.1.1911
openSUSE Tumbleweed 20200103
Fedora 31 (Server Edition)

**Comment by deokershesh** [comment]
Thanks ewmadrid, we are working on it, we will provide/host the Linux binaries shortly.

**Comment by rhtpandeyIN** [comment]
Hi Eric @ewmadrid ,

I have created the binaries for the Linux platform, but it will take some time to push them to PyPi org. As we need to test the new binaries for all the platforms, and after that we will host them into PyPi.

If you need them urgently or wants to try the new binaries ASAP, then please share your email ID with us, we will share the Linux binary with you. Try that in your environment and let us know the output. 

Thanks 
Rohit.

**Comment by rhtpandeyIN** [comment]
Hi @ewmadrid , I have added the Linux binary for Python 3.8.1 version on Pypi org, now you can install the module by using below command. It should work for Python 3.8.1 on Linux platform. (Windows binary will come soon).
```
pip3 install ifxpy
```

**Comment by deokershesh** [comment]
@ewmadrid, Hope you are able to use successfully now. Closing the ticket.

**Comment by ewmadrid** [comment]
Thank you very much, it is working great!

**Comment by valtanders** [comment]
> 
> 
> Hi @ewmadrid , I have added the Linux binary for Python 3.8.1 version on Pypi org, now you can install the module by using below command. It should work for Python 3.8.1 on Linux platform. (Windows binary will come soon).
> 
> ```
> pip3 install ifxpy
> ```

hello! any idea when it's going to be available for windows?

**Comment by rhtpandeyIN** [comment]
@valtanders , We are working on that, but there are some issue with Python 3.8.X versions. We will let you know once it is done. It will be very soon though. 

Regards:
Rohit

**Comment by gitThilanka** [comment]
> @valtanders , We are working on that, but there are some issue with Python 3.8.X versions. We will let you know once it is done. It will be very soon though.
> 
> Regards:
> Rohit

HI rhtpandeyIN any progress in windows binary ?
